README.md

```diff
@@ -104,7 +106,7 @@ RTR-Server using the rtrclient command line tool:
 
     bin/rtrclient tcp rpki-validator.realmv6.org 8282
 
-rpki-validator.realmv6.org is an open RTR-Server instance for testing
+`rpki-validator.realmv6.org` is an open RTR-Server instance for testing
 purposes, which runs the RIPE Validator. It listens on port 8282 and
```

Comment:
maybe change to 8283 here to use validator 3 instead of 2?

Reply:
no, not now.